src/TorchSharp/TorchVision/models/Utils.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
+
+using System;
+
+using static TorchSharp.torch;
+
+namespace TorchSharp
+{
+    public static partial class torchvision
+    {
+        public static partial class models
+        {
+            internal static partial class _utils
+            {
+                /// <summary>
+                /// This function is taken from the original tf repo.
+                /// It ensures that all layers have a channel number that is divisible by 8
+                /// It can be seen here:
+                /// https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet/mobilenet.py
```

Comment:
Please add a reference to the TF license and copyright notice from here.

Reply:
Added references. Also fix wrong mentions to torchaudio with torchvision.